cpp/ql/test/library-tests/dataflow/dataflow-tests/dispatch.cpp

```diff
@@ -20,29 +20,29 @@ struct Middle : Top {
 // This class has all the behavior suggested by the function names.
 struct Bottom : Middle {
   int isSource1() override { return source(); }
-  void isSink(int x) override { sink(x); }
+  void isSink(int x) override { sink(x); } // $ ir=33:18 ir=41:17 ir=69:15 ir=73:14 ir=81:13 MISSING: ast=33:18 ast=41:17 ast=69:15 ast=73:14 ast=81:13
   int notSource1() override { return 0; }
   void notSink(int x) override { }
 };
 
 void VirtualDispatch(Bottom *bottomPtr, Bottom &bottomRef) {
   Top *topPtr = bottomPtr, &topRef = bottomRef;
 
-  sink(topPtr->isSource1()); // flow [NOT DETECTED by AST]
-  sink(topPtr->isSource2()); // flow [NOT DETECTED by AST]
-  topPtr->isSink(source()); // flow [NOT DETECTED by AST]
+  sink(topPtr->isSource1()); // $ ir MISSING: ast
+  sink(topPtr->isSource2()); // $ ir MISSING: ast
+  topPtr->isSink(source()); // causing a MISSING for ast
```

Comment:
Is there a reason this one is not an inline expectation comment (and similarly for the others below this one)?

Reply:
Yes indeed, as the sink function is not called here, but in the implementation of `Bottom`, this is not the correct target for the inline expectation comment. However I still wanted to annotate the caller here with something. Let me know if/how this could be made more obvious.

Reply:
Ah, now I understand. Thanks!